### PR TITLE
scikit-learn 0.22 compatibility

### PR DESCRIPTION
**Pull request by zzz4zzz**
Adjust ELI5 to scikit-learn 0.22.

### CODE REVIEW

**Comment by zzz4zzz** [comment]
Failed tests are not related to new sklearn version.

File: eli5/sklearn/text.py [hunk 1]
```diff
@@ -59,6 +59,10 @@ def _get_doc_weighted_spans(doc,
     if hasattr(vec, 'get_doc_weighted_spans'):
         return vec.get_doc_weighted_spans(doc, feature_weights, feature_fn)
 
+    try:
+        from sklearn.feature_extraction.text import _VectorizerMixin as VectorizerMixin
+    except ImportError:  # Changed in scikit-learn 0.22
+        from sklearn.feature_extraction.text import VectorizerMixin  # type: ignore
```
[the remark this answers is not in the document]
Comment (zzz4zzz) [reply eli5/sklearn/text.py]: Yes, that's right. I have just moved import to the top of file.

File: eli5/sklearn/text.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
 from __future__ import absolute_import
 from typing import Any, Union, Callable, Dict, List, Optional, Set, Tuple
 
-from sklearn.feature_extraction.text import VectorizerMixin  # type: ignore
 from sklearn.pipeline import FeatureUnion  # type: ignore
+try:
+    from sklearn.feature_extraction.text import _VectorizerMixin as VectorizerMixin
+except ImportError:  # Changed in scikit-learn 0.22
```
Comment (lopuhin) [inline eli5/sklearn/text.py]: FWIW it still works in sklearn 0.22, but it's already deprecated (to be removed in 0.24), so good that we are ready 👍
